includes/class-woothemes-sensei-frontend.php

```diff
@@ -1148,6 +1152,33 @@ public function sensei_no_permissions_main_content() {
 		} // End While Loop
 	} // End sensei_no_permissions_main_content()
 
+
+    /**
+     * sensei_course_category_order_main_content function
+     *
+     * Sort the main query on custom taxonomy pages to pass posts in
+     * 'menu_order', because Sensei allows users to set the order of
+     * Courses under Course > Course Order.
+     *
+     * @since 1.9.0
+     * @access public
+     * @param object $query
+     * @return object $query
+     */
+
+    public function sensei_course_category_order_main_content($query) {
+
+        if ( is_tax() ) {
```

Comment:
@mahangu too broad. Please specify the taxonomy as we don't want this to fire for all taxonomy queries.